Fix #83940

hadzhiyski opened the pull request:
Supported languages: de, en, es, fr, hu, it, pt-br, ru

<!-- Thank you for submitting a Pull Request. Please: 
* Read our Pull Request guidelines:
  https://github.com/Microsoft/vscode/wiki/How-to-Contribute#pull-requests.
* Associate an issue with the Pull Request.
* Ensure that the code is up-to-date with the `master` branch.
* Include a description of the proposed changes and how to test them. 
-->

This PR fixes #83940

msftclas commented:
[![CLA assistant check](https://cla.opensource.microsoft.com/pull/badge/not_signed)](https://cla.opensource.microsoft.com/microsoft/vscode?pullRequest=84414) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/vscode?pullRequest=84414) before we can accept your contribution.<br/><br/>:x: hadzhiyski [sign now](https://cla.opensource.microsoft.com/microsoft/vscode?pullRequest=84414)<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.opensource.microsoft.com/check/microsoft/vscode?pullRequest=84414) it.</sub>

jaqra commented on build/win32/i18n/messages.en.isl:
```diff
@@ -5,4 +5,5 @@ AssociateWithFiles=Register %1 as an editor for supported file types
-SourceFile=%1 Source File
\ No newline at end of file
+SourceFile=%1 Source File
+OpenWithCodeContextMeny=Open w&ith %1
```
```suggestion
OpenWithCodeContextMenu=Open w&ith %1
```

jaqra commented:
https://github.com/microsoft/vscode/blob/89659aa4b3fa350e948f3bbc54caf9da8a78be33/build/win32/i18n/messages.tr.isl#L8

You can add

```
OpenWithCodeContextMeny=%1 İle Aç
```

But i am not sure which key must be set as shortkey

hadzhiyski commented:
> https://github.com/microsoft/vscode/blob/89659aa4b3fa350e948f3bbc54caf9da8a78be33/build/win32/i18n/messages.tr.isl#L8
> 
> You can add
> 
> ```
> OpenWithCodeContextMeny=%1 İle Aç
> ```
> 
> But i am not sure which key must be set as shortkey

Thanks. I will try to research what is the correct translation for all supported languages

jaqra commented:
It is correct. It is my main language. You can also see from

https://github.com/microsoft/vscode/blob/89659aa4b3fa350e948f3bbc54caf9da8a78be33/build/win32/i18n/messages.tr.isl#L2

jaqra commented on build/win32/i18n/messages.ru.isl:
```diff
@@ -1,8 +1,9 @@
-Other=������:
-SourceFile=�������� ���� %1
\ No newline at end of file
+AddContextMenuFiles=�������� �������� "������� � ������� %1" � ����������� ���� ����� ���������� Windows
```
Lastly i want to say. It seems your editor changes charsets of files. May this breaks

jaqra replied:
Sorry for one by one comment 😕

hadzhiyski replied:
I will look into it later today

hadzhiyski replied:
I am using VS Code to do the changes, but perhaps the encodings are not ok on my PC

hadzhiyski commented:
Translations are added for all supported languages and the encoding issues are fixed. Please review

hadzhiyski commented:
Is that what you mean? I deleted the empty lines on end of files, because that was the original look of the files. I added them now. Perhaps I do not have proper file formatting settings and I am really sorry about this and I will fix it. It is my first PR in vscode repository.

joaomoreno commented:
Not yet... Notice here, all these lines are marked as changed, yet there is no reason for you to have changed them. There must still be some line-ending/encoding change:

![image](https://user-images.githubusercontent.com/22350/68581856-96749c80-0479-11ea-8c1c-85acf5223eb4.png)

jaqra commented on build/win32/i18n/messages.de.isl:
```diff
@@ -1,8 +1,9 @@
 [CustomMessages]
```
You can try to copy this line's enter char and paste other lines for enter character

hadzhiyski replied:
I think it's ok now. The huge diff comes from
![image](https://user-images.githubusercontent.com/20074286/68586090-8c0fce00-048c-11ea-8319-9943fc5b8e1d.png)

However, <F6> and <C3><B6> is basically the same.
![image](https://user-images.githubusercontent.com/20074286/68586151-a8ac0600-048c-11ea-9046-436e66da98ae.png)

jaqra replied:
May you should set charset as ANSI

hadzhiyski replied:
i think i managed to fix this

joaomoreno commented:
Great job, thanks!

hadzhiyski commented:
Thank you all for the support
